components/formstack/actions/create-form/create-form.mjs

```diff
@@ -0,0 +1,73 @@
+import formstack from "../../formstack.app.mjs";
+
+export default {
+  type: "action",
+  key: "formstack-create-form",
+  name: "Create Form",
+  description: "Create a new form in your account. [See docs here](https://formstack.readme.io/docs/form-post)",
+  version: "0.0.1",
+  props: {
+    formstack,
+    name: {
+      label: "Name",
+      description: "The form name",
+      type: "string",
+    },
+    fields: {
+      label: "Fields",
+      description: "Array of Field resources",
```

Comment:
Can you expand a bit more the description here, so the user can get a better idea on how to input these values?, maybe we can change this to an array of strings? `type: "string[]"`.
Isn't this the same as the property in `create-form-submission` action component?

Reply:
> Isn't this the same as the property in create-form-submission action component?

Isn't the same prop, one is just the field id and another is the field with values.